src/hooks/useFilter.ts

```diff
@@ -0,0 +1,33 @@
+import { useFilterContext } from "@/context/filter";
+import { Entries } from "@/types/interfaces";
+import { useCallback, useEffect, useState } from "react";
+
+export const useSelectedEntries = (entries: Entries) => {
+  const [selectedEntries, setSelectedEntries] = useState<Entries>(entries);
```

Comment:
The hook should reflect the name of the file. hence it should be `useFilter`

Reply:
changed